src/sentry/web/api.py

```diff
@@ -36,3 +36,8 @@ def crossdomain_xml(request, project_id):
     response["Content-Type"] = "application/xml"
 
     return response
+
+
+@cache_control(max_age=3600, public=True)
+def not_found(request):
+    return HttpResponse(status=404)
```

Comment:
Does this return 404 on the catch-all case? I'm not sure if we should change it.

Reply:
This is just a handler function. Catch-all is not changed, merely the path stated [here](https://github.com/getsentry/sentry/pull/50716/files#diff-e5159fd2d611249933f89edbe3bfb3c4fcc01620b1e65fb3ff3100d1983ba838R106): `^api/relay/.*$`